src/main/java/io/muehlbachler/bswe3b/CalculatorService.java

```diff
@@ -2,12 +2,31 @@
 
 import org.springframework.stereotype.Service;
 
-// FIXME: add javadoc
+
+/**
+ * Diese Klasse bietet Dienste für einfache Berechnungen.
+ */
 @Service
 public class CalculatorService {
-    // FIXME: add javadoc
+
+    /**
+     * Diese Methode teilt zwei Ganzzahlen.
+     *
+     * @param a Der Dividend.
+     * @param b Der Divisor.
+     * @return Das Ergebnis der Division.
+     */
     public int divide(final int a, final int b) {
-        // FIXME: implement
-        return -1;
+        // Prüfen, ob der Divisor nicht null ist, um eine Division durch null zu verhindern.
```

Comment:
this is not necessary as it's pretty clear from the javadoc of the method and the code itself.